image/v1/types.go

```diff
@@ -159,9 +165,20 @@ type ImageStreamList struct {
 // +genclient:method=Layers,verb=get,subresource=layers,result=github.com/openshift/api/image/v1.ImageStreamLayers
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
-// ImageStream stores a mapping of tags to images, metadata overrides that are applied
+// An ImageStream stores a mapping of tags to images, metadata overrides that are applied
```

Comment:
have the godoc rules(guidelines?) been loosed to allow articles to preface the type/field/function name?

Reply:
These are turned into user facing doc, like api type field docs.  So they should be targeted at humans.

Reply:
i mean, i agree, but we also have things like golint that complain about this stuff, right?

Reply:
Golint should never complain on types.go on that particular error.